src/components/PageReady/config/Context.js

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+
+const Context = React.createContext({ data: null });
```

Comment:
i don't see much use in having this in a separate file

Reply:
cool refactored to have context in the same `PageReady` component